DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs

```diff
@@ -18,24 +19,30 @@ public PostLearningAssessmentViewModel(PostLearningAssessment postLearningAssess
             CourseTitle = postLearningAssessment.CourseTitle;
             SectionName = postLearningAssessment.SectionName;
             PostLearningAssessmentPath = postLearningAssessment.PostLearningAssessmentPath;
+            PostLearningScore = postLearningAssessment.PostLearningScore;
+            PostLearningAttempts = postLearningAssessment.PostLearningAttempts;
+            PostLearningPassed = postLearningAssessment.PostLearningPassed;
             PostLearningLocked = postLearningAssessment.PostLearningLocked;
+            CustomisationId = customisationId;
+            SectionId = sectionId;
+        }
 
-            if (postLearningAssessment.PostLearningAttempts == 0)
-            {
-                AssessmentStatus = "Not attempted";
-            }
-            else
+        public string GetAssessmentStatus()
```

Comment:
This doesn't need to be public. It can be a private method which you use to set the assessment status in the constructor. By giving it appropriate parameters you can store fewer variables while still keeping the constructor simple. Does that make sense? 🙂

Reply:
Ah I see, sorry for the misunderstanding 🤦‍♂️